Gemfile

```diff
@@ -82,6 +82,7 @@ end
 
 group :test do
   gem 'faker'
+  gem 'faker-medical'
```

Comment:
So this gem randomly generates different fake data on each rspec run?  Is it possible that this could introduce instability by failing intermittently on some specific piece of test data?